doc/en/developer/source/policies/community-process.rst

```diff
@@ -41,6 +41,17 @@ The determining factor for what release a feature should be included in is based
 
 New features may be back-ported to the stable series (if technically feasible) after being tried out on master for a month.
 
+Adding fixes
+^^^^^^^^^^^^
+
+During the release cycle community members contribute fixes to be included, and backported, to be included in subsequent releases. 
+
+#. Each fix requires an issue tracker entry, to be included in the release notes
+#. Each fix must be applied to the master branch, and then back ported.
+#. While a release may be held for a "blocking" issue this is determined by discussion on the developer email list.
+
+Please respect our release volunteers, we stop back porting fixes just prior to release in order to perform quality assurance.
```

Comment:
"just prior" means nothing. Let's make it the day before the release so that CITE tests already contain all the changes needed.